README.md

```diff
@@ -40,6 +40,45 @@ api.add_resource("/widgets", WidgetListView, WidgetView)
 
 Documentation is available at https://flask-resty.readthedocs.io/.
 
+## Running locally
+
+Ideally, create a virtual env for local development, using your tool of choice, e.g.:
+
+```sh
+pyenv virtualenv 3.8.3 flask-resty
+pyenv local flask-resty
+```
+
+Install dependencies:
+
+```sh
+# optionally include [jwt] for the full test suite, but requires cryptography,
+# and it's dependent binary libraries
+pip install -e .[testing,docs]
+```
+
+### Running tests
+
+Tests can be run for your current enviroment by running pytest directly: `pytest`.
+
+Or run `tox` to run the fulle suite of lint checks and tests against flask-resty's supported
+environments.
+
+### Building the docs
+
+Run sphinx to build a copy of the docs locally
+
+```sh
+sphinx-build docs/ docs/_build
+```
+
+For auto rebuilds and watching, `sphinx-autobuild` can be used:
+
+```sh
+pip install sphinx-autobuild
+sphinx-autobuild --open-browser docs/ docs/_build -z flask_resty -s 2
+```
```

Comment:
suggestion: Probably better to just document one way to do things. Most people will want autobuild. Also, you can use tox for this:

```
tox -e watch-docs
```

which means you don't need to install anything extra yourself.

Here's how it's documented in marshmallow: https://marshmallow.readthedocs.io/en/stable/contributing.html#documentation